doc/speed-manual.tex

```diff
@@ -363,9 +409,9 @@ \subsection{Job Submission Basics}
 provided to the scheduler option, \option{-pe smp}), though, obviously, every
 program has its own unique execution framework.
 
-Please note that jobs that expect to touch data-input and -output files
+Please note that jobs that expect to touch data-input and data-output files
 more than once should make use of \api{TMPDIR}, a scheduler-provided working
-space (this is quite generous, close to 1~TB in size). \api{TMPDIR} is created
+space (this is quite generous, close to ~1TB in size). \api{TMPDIR} is created
```

Comment:
`1~TB` is actually correct, here is the ~ is a non-breaking space and not an "approximate" sign.